pkgs/tools/X11/xpra/default.nix

```diff
@@ -189,6 +189,7 @@ in buildPythonApplication rec {
   # append module paths to xorg.conf
   postInstall = ''
     cat ${xorgModulePaths} >> $out/etc/xpra/xorg.conf
+    cat ${xorgModulePaths} >> $out/etc/xpra/xorg-uinput.conf
```

Comment:
```suggestion
    cat ${xorgModulePaths} >> $out/etc/xpra/xorg.conf
    ln -s $out/etc/xpra/xorg.conf $out/etc/xpra/xorg-uinput.conf
```

Reply:
Those two files contain different configurations (for non-uinput and uinput). We are just appending `xorgModulePaths` to the existing contents.